---
source: tests/suite/check.rs
assertion_line: 35
expression: Snapshot(context)
---
Error: Semantic error
   ╭─[tests/check/bad_class_shadowing_6.xi:3:5]
   │
 2 │     a: int
   ·     ┬  
   ·     ╰── Previous definition here
 3 │     a() {}
   ·     ┬  
   ·     ╰── Name already bound in environment
───╯

